### PR TITLE
Regenerate SDK with finalized SDK generator and write integration tests

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary
Sorry for the huge PR....

This pull request addresses all generator issues in the Unity SDK and regenerates all services based on the latest openapi3 specs. Integration tests for all services were written using UnityTestRunner.

### CODE REVIEW

File: Scripts/Services/Assistant/V1/Model/DialogNode.cs [hunk 1]
```diff
@@ -199,7 +199,7 @@ public class DigressOutSlotsValue
         /// - It must be no longer than 1024 characters.
         /// </summary>
         [JsonProperty("dialog_node", NullValueHandling = NullValueHandling.Ignore)]
-        public string DialogNodeId { get; set; }
+        public string _DialogNode { get; set; }
```
Comment (mamoonraja) [inline Scripts/Services/Assistant/V1/Model/DialogNode.cs]: why is it `_DialogNode` now?

File: Scripts/Services/CompareComply/V1/Model/FeedbackDeleted.cs [hunk 2]
```diff
@@ -17,22 +17,22 @@
 
 using Newtonsoft.Json;
 
-namespace IBM.Watson.Discovery.V1.Model
+namespace IBM.Watson.CompareComply.V1.Model
```
Comment (mamoonraja) [inline Scripts/Services/CompareComply/V1/Model/FeedbackDeleted.cs]: Maybe I am missing it but I am not seeing the `Discovery` Model, are we getting rid of it?

File: Scripts/Services/Discovery/V1/Model/TrainingExample.cs [hunk 3]
```diff
@@ -25,17 +25,17 @@ namespace IBM.Watson.Discovery.V1.Model
     public class TrainingExample
     {
         /// <summary>
-        /// Gets or Sets DocumentId
+        /// The document ID associated with this training example.
```
Comment (mamoonraja) [inline Scripts/Services/Discovery/V1/Model/TrainingExample.cs]: Ignore the comment on CNC, here it is.

File: Scripts/Services/Assistant/V1/Model/DialogNode.cs (hunk 1 above)
Comment (mediumTaj) [reply Scripts/Services/Assistant/V1/Model/DialogNode.cs]: a model cannot have a property with the same name (in this case DialogNode.DialogNode). That is reserved for the constructor. In the generator we append `_` to this. When it was handwritten I just added `Id`

File: Scripts/Services/CompareComply/V1/Model/FeedbackDeleted.cs (hunk 2 above)
Comment (mediumTaj) [reply Scripts/Services/CompareComply/V1/Model/FeedbackDeleted.cs]: Hmm the discovery models are there  - this might be some weird renaming thing

**Pull request merged**